packages/admin/src/pages/build-application/[applicationId]/[sectionId]/[questionId]/unpublished-preview/index.page.tsx

```diff
@@ -73,11 +82,22 @@ const getServerSideProps = async ({
       ? `/build-application/${applicationId}/${sectionId}/${nextQuestionId}/unpublished-preview`
       : null;
 
+    const prevQuestionId = getPrevQuestionId(
+      currentQuestion.questionId,
+      questionIds
+    );
+
+    //determine backHref value
+    isFirstQuestion(currentQuestion.questionId, questionIds)
+      ? (backHref = `/build-application/${applicationId}/preview`)
+      : (backHref = `/build-application/${applicationId}/${sectionId}/${prevQuestionId}/unpublished-preview`);
+
```

Comment:
Will be easier to read if we make `backHref` a const and assign it here instead of reassigning within the ternary